.github/ISSUE_TEMPLATE/installation_issue.yaml

```diff
@@ -0,0 +1,61 @@
+name: Installation Issue
+description: Report issues installing the pandas library on your system
+title: "BUILD: "
+labels: [Build, Needs Triage]
+
+body:
+  - type: checkboxes
+    id: checks
+    attributes:
+      options:
+        - label: >
+            I have read the [installation guide](https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html#installing-pandas).
+          required: true
+  - type: input
+    id: operating-system
+    attributes:
+      label: Operating System
+      description: >
+        Please provide the operating system you are running.
+      placeholder: >
+        e.g. Windows 10, macOS 11 Big Sur, Ubuntu 18.04 LTS
+    validations:
+      required: true
+  - type: input
+    id: platform
+    attributes:
+      label: Platform
+      description: >
+        Please provide the hardware platform you are running.
+      placeholder: >
+        e.g. x86-64, ARM64 (M1)
```

Comment:
Maybe add some code showing how to find the hardware platform. (e.g. ``import platform; platform.machine()``)?

Reply:
Thanks for the tip. Found that `platform.platform()` is even more comprehensive so added that instead.